components/wmtk_components/extract_subset/internal/extract_subset_2d.cpp

```diff
@@ -0,0 +1,130 @@
+#include "extract_subset_2d.hpp"
+
+wmtk::TriMesh extract_subset_2d(
+    const std::vector<Eigen::Vector2d>& points,
+    Eigen::MatrixXi& triangles, std::vector<size_t> tag){
+
+    int nb_vertex = points.size();
```

Comment:
Should be `const` and it should be a `size_t` just like `points.size()`. If you want it to be an `int` for whatever reason, you should convert it explicitly because `size_t` to `int` is a narrowing conversion and it will produce a warning.